molgenis/ds-tidyverse
# feat: if_else

## To test:
See instructions on corresponding client-side package [#41](https://github.com/molgenis/ds-tidyverse-client/pull/41)
